### ISSUE TITLE
502 Error

### ISSUE THREAD

**Issue by MrSleeps** [alexander0042/pirate-weather-ha#42]
For the last few months I've been getting the odd 502 error which was on the odd request but for the last month every request chucks  back a 502 error.

Log:

`Logger: custom_components.pirateweather.weather
Source: custom_components/pirateweather/weather.py:278
Integration: pirateweather ([documentation](https://github.com/alexander0042/pirate-weather-ha), [issues](https://github.com/alexander0042/pirate-weather-ha/issues))
First occurred: 20:42:34 (1 occurrences)
Last logged: 20:42:34
Unable to connect to PirateWeather. 502 Server Error: Bad Gateway for url: https://api.pirateweather.net/forecast/**API-KEY**/50.82639,-0.16587?units=si`

**Comment by alexander0042** [comment]
Quick update: I pushed a new version of the API ("V1.1.4") this morning since once I noticed the grid edge errors, I couldn't just leave them there. No idea who's out there requesting forecasts for the North and South poles. but at least they're working now!

This seems to have solved most of the 502 errors that I was seeing: 
![image](https://user-images.githubusercontent.com/26213938/159557814-c662b53d-ab34-4210-b494-4d52759efa42.png)

I'm not sure if this will fix your issue, but at least it should make it easier to diagnose, since it quiets down the noise of other errors in the logs
